### DISCUSSION TITLE
Styling

### DISCUSSION THREAD

**Discussion by jabez-du-preez** [leandrowd/react-responsive-carousel#710] [General]
How do I change the color of the arrows? Right now I see the default color is white. And I have a white background already, so I cannot see the arrows to switch slides. 

Please help.

**Comment by adriancoliba** [comment]
If you inspect the element, there is a border color to the prev and next arrow: 
.carousel .control-prev.control-arrow:before {
    border-right: 8px solid #fff;
}
.carousel .control-next.control-arrow:before {
    border-left: 8px solid #fff;
}

You could add these classes to your CSS file and change the color (in my case, I had to use !important keyword):
.carousel .control-prev.control-arrow:before {
    border-right: 8px solid black !important;
}
.carousel .control-next.control-arrow:before {
    border-left: 8px solid !important;
}
